bip39-lib/src/mingwMain/kotlin/cash/z/ecc/android/random/SecureRandom.kt

```diff
@@ -0,0 +1,27 @@
+package cash.z.ecc.android.random
+
+import kotlinx.cinterop.CPointer
+import kotlinx.cinterop.UByteVar
+import kotlinx.cinterop.addressOf
+import kotlinx.cinterop.convert
+import kotlinx.cinterop.usePinned
+import platform.windows.BCRYPT_USE_SYSTEM_PREFERRED_RNG
+import platform.windows.BCryptGenRandom
+import platform.windows.CMC_STATUS_SUCCESS
+
+actual class SecureRandom {
+    actual fun nextBytes(bytes: ByteArray) {
+        val result = bytes.usePinned {
+            @Suppress("UNCHECKED_CAST")
```

Comment:
What errors occur if this @Suppress is removed? Are there consequences to making the value nullable instead of suppressing the unchecked cast?

Reply:
```
> Task :bip39-lib:compileKotlinMingwX64 FAILED
e: warnings found and -Werror specified
w: file:///Users/lucaspinazzola/Projects/kotlin-bip39/bip39-lib/src/mingwMain/kotlin/cash/z/ecc/android/random/SecureRandom.kt:17:33 Unchecked cast: CPointer<ByteVar /* = ByteVarOf<Byte> */> to CPointer<UByteVar /* = UByteVarOf<UByte> */>

Execution failed for task ':bip39-lib:compileKotlinMingwX64'.
> Compilation finished with errors
```

Reply:
it's supposed to be a UByte not a Byte for mingw. The suppress isn't for null-ablity